README.md

```diff
@@ -91,12 +91,12 @@ expected location. Also, no more _reshim_ needed upon `npm install`.
 
 ### Setup
 
-Install this plugin and run the automatic setup: `asdf direnv setup <shell>`.
+Install this plugin and run the automatic setup.
 You can run this command for all of your preferred shells `bash`/`fish`/`zsh`.
```

Comment:
Would it make sense to mention here that `--version` can be `system` / `latest` / `<direnv-release-version>` ?

Reply:
Good idea, done!